### DISCUSSION TITLE
504 error when performing specific operations

### DISCUSSION THREAD

**Discussion by MomentQYC** [minio/minio#17658] [Q&A] [answered]
## Expected Behavior

Normal display of content and normal execution of operations.

## Current Behavior

Display 504 error after loading for some time.
## Steps to Reproduce (for bugs)

1. Visit `identity/users` (or `identity/groups`, `policies`) in the browser
2. Error 504 after loading for a while.

Here is information that may be helpful:
![image](https://github.com/minio/minio/assets/62551256/d7f4469d-ef0f-456b-b7f3-34d3cd879b21)

## Context

Unable to perform users-related operations.

## Regression

No, personally it's a long standing problem.
## Your Environment

* Version used (`minio --version`): `RELEASE.2023-07-07T07-13-57Z (commit-id=e20aab25ec26ef3b1e44eac1f6a01f15f043ab42)`
* Server setup and configuration: deb package
```
MINIO_VOLUMES="/path/to/data"
MINIO_ROOT_USER=***
MINIO_ROOT_PASSWORD=***
MINIO_DOMAIN=***
```
* Operating System and version (`uname -a`): `5.4.0-148-generic #165-Ubuntu SMP Tue Apr 18 08:53:12 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux`

**Comment by harshavardhana** [comment]
Make sure you have enabled websockets support in your load balancer talking to Console UI.

**Reply by MomentQYC** [reply]
It didn't work.

**Comment by MomentQYC** [comment]
As a side note, direct access without logging in shows the returned information, but logging in gives you an error. This problem occurs even when curl 127.0.0.1 carries the cookie.
That is, it's not an Nginx issue.

**Comment by MomentQYC** [comment]
As well I found a very strange error report that seems to happen at the same time as the request.
```
 API: SYSTEM()
 Time: 06:10:47 UTC 07/15/2023
 DeploymentID: 33885295-cea1-4c3a-b98c-26e14f87dc0b
 Error: Failure in periodic refresh for IAM (took 0.06s): madmin: invalid encryption algorithm ID (*errors.errorString)
        2: internal/logger/logger.go:258:logger.LogIf()
        1: cmd/iam.go:450:cmd.(*IAMSys).watch()
```

**Reply by harshavardhana** [reply]
This is because you have a setup issue here @MomentQYC

**Comment by MomentQYC** [comment, marked as answer]
It seems to be related to KES, resetting the configuration solved the problem.
